### ISSUE TITLE
Accessibility problems

### ISSUE THREAD

**Issue by vertigo7x** [Teradata/covalent#874]
### Do you want to request a *feature* or report a *bug*?
bug/feature

#### Bug Report
Doing some research about accessibility, I found accessibility issues using the Axe Accessibility Testing Tool.
- Buttons must have discernible text: some buttons doesn't have a label nor an aria-label attribute. Components affected:  
  - layout: td-menu-button
  - search-box: td-search-icon
  - paging: td-paging-bar-first-page, td-paging-bar-prev-page, td-paging-bar-prev-page, td-paging-bar-next-page, td-paging-bar-last-page
  - messages: the toggle visibility button
- All th element and elements with role=columnheader/rowheader must data cells which it describes: the dynamic tables have a table for the headers, but this table is empty of data. Components affected:
  - tables
- lang attribute must have a valid value: only valid languages can be specified by the lang attribute. Components affected:
  - highlight

#### What is the expected behavior?
Pass accessibility validation

#### What is the motivation / use case for changing the behavior?
Maybe some projects don't need to be accessible/valid but as Covalent uses components from Material, that are accessible. Why not make everything accessible?

#### Which version of Angular and Material, and which browser and OS does this issue affect?
Covalent 1.0.0-beta.7

##### Other information
Some changes are pretty easy, some might break the current implementation of components.

**Comment by kyleledbetter** [comment]
Thanks for reporting. a11y & testing with http://pa11y.org/ will be a big focus as we move closer to stable

**Comment by owilliams320** [comment]
this component is deprecated please use https://material.angular.io/components/table
